src/hammer-vlsi/hammer_utils/spice_utils.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+#  spice_utils.py
+#  Misc Verilog utilities
+#
+#  See LICENSE for licence details.
+
+import re
+from typing import Tuple, Optional, List, Set, Dict
+from enum import Enum
+
+class SpiceUtils:
+
+    # Multilines in spice start with a + character
+    multiline_pattern = re.compile("(?:\s*\n\+)+\s*", flags=re.DOTALL)
```

Comment:
Minor nitpicks: if these are intended to be constants I would make them uppercase

Reply:
fixed